pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -305,23 +305,36 @@ func maybeRetryDownload(ctx *downloaderContext,
 			status.Key())
 		return
 	}
-	status.LastErr = ""
-	status.LastErrTime = time.Time{}
-	status.RetryCount += 1
```

Comment:
Need to add back this line to increase RetryCount (and it should not be inside the ClearErrorInfo but right here)

Reply:
Sure.

Reply:
I don't see the RetryCount having eeb moved back. Is there a git push coming?

Reply:
missed, added it back.